src/GitLabHealth-Model-Analysis-Tests/CodeDeletionByMRMetricTest.class.st

```diff
@@ -1,98 +1,98 @@
-			 author: user;
-			 project: project1;
-			 created_at: '2024-09-05T10:24:11.803+02:00' asDateAndTime;
-			 merged_at: '2024-09-05T11:24:11.803+02:00' asDateAndTime;
-			 state: #merged;
-			 title: '205 feat do something') }.
-
-	glhImporter commits: {
-			(GLHCommit new
-				 id: 1;
-				repository: project1 repository;
-			 created_at: '2024-09-05T10:24:11.803+02:00' asDateAndTime;
-				 additions: 7;
-				 commitCreator: user;
-				 deletions: 5).
-			(GLHCommit new
-				 id: 2;
-				repository: project1 repository;
-			 created_at: '2024-09-05T10:34:11.803+02:00' asDateAndTime;
-				 additions: 3;
-				 commitCreator: user;
-				 deletions: 5) }.
-
-	codeDeletion := CodeDeletionByMRMetric new
-		                user: user;
-		                glhImporter: glhImporter;
-		                setPeriodSince: '2024-09-04' until: '2024-09-04';
-		                over: Week.
-
-	"When"
-	result := codeDeletion calculate.
-
-	"Then"
-	self assert: result equals: 10
-]
-
-{ #category : #tests }
-CodeDeletionByMRMetricTest >> testCalculateNoCommits [
-
-	| result glhImporter codeDeletion |
-	"Given"
-	glhImporter := GLPHImporterMock new.
-	glhImporter mergeRequests: { (GLHMergeRequest new
-			 author: user;
-			 created_at: createdAt ;
-			 state: #opened;
-			 title: '205 feat do something') }.
-
-	codeDeletion := CodeDeletionByMRMetric new
-		                user: user;
-		                glhImporter: glhImporter;
-		                setPeriodSince: since until: until;
-		                over: Week.
-
-	"When"
-	result := codeDeletion calculate.
-
-	"Then"
-	self assert: result equals: 0
-]
-
-{ #category : #tests }
-CodeDeletionByMRMetricTest >> testCalculateNoMergeRequests [
-
-	| result glhImporter codeDeletion |
-	"Given"
-	glhImporter := GLPHImporterMock new.
-	glhImporter mergeRequests: {  }.
-
-	codeDeletion := CodeDeletionByMRMetric new
-		                user: user;
-		                glhImporter: glhImporter;
-		                setPeriodSince: since until: until;
-		                over: Week.
-
-	"When"
-	result := codeDeletion calculate.
-
-	"Then"
-	self assert: result equals: 0
-]
+"
+A CodeDeletionByMRMetricTest is a test class for testing the behavior of CodeDeletionByMRMetric
+"
+Class {
+	#name : #CodeDeletionByMRMetricTest,
+	#superclass : #UserMetricTest,
+	#category : #'GitLabHealth-Model-Analysis-Tests'
+}
+
+{ #category : #tests }
+CodeDeletionByMRMetricTest >> testCalculate [
+
+	| result glhImporter  codeDeletion |
+
+
+	glhImporter := GLPHImporterMock new.
+	glhImporter mergeRequests: { (GLHMergeRequest new
+			 author: user;
+			 project: project1;
+			 created_at: '2024-09-05T10:24:11.803+02:00' asDateAndTime;
+			 merged_at: '2024-09-05T11:24:11.803+02:00' asDateAndTime;
+			 state: #merged;
+			 title: '205 feat do something') }.
+
+	glhImporter commits: {
+			(GLHCommit new
+				 id: 1;
+				repository: project1 repository;
+			 created_at: '2024-09-05T10:24:11.803+02:00' asDateAndTime;

```

Comment:
please consider date format, used instance variable declare in super class setup